### DISCUSSION TITLE
F-22 and Su-57

### DISCUSSION THREAD

**Discussion by RaptorDad86** [Khopa/dcs_liberation#2386] [Q&A] [answered]
Hello,

Currently I'm unable to use the F-22 and Su-57 with the 5.2 build. I make sure to check the boxes prior to starting a new campaign but they don't show up. I was able to use them in previous builds and I verified they are currently installed. 

Any help would be greatly appreciated!

**Comment by Starfire13** [comment, marked as answer]
It's possible the mods have been updated since. They will only work with the version if the mod that was current back when 5.2 was developed.
